libr/util/num.c

```diff
@@ -277,19 +279,15 @@ R_API ut64 r_num_math(RNum *num, const char *str) {
 #endif
 }
 
-R_API int r_num_is_float (struct r_num_t *num, const char *str) {
+R_API int r_num_is_float(struct r_num_t *num, const char *str) {
 	// TODO: also support 'f' terminated strings
 	return (strchr (str, '.') != NULL)? R_TRUE:R_FALSE;
 }
 
-R_API double r_num_get_float (RNum *num, const char *str) {
-	typedef union {
-		ut64 n;
-		double d;
-	} dualtype;
-	dualtype dual;
-	dual.n = (ut64)r_num_get (NULL, str);
-	return (double)dual.d;
+R_API double r_num_get_float(struct r_num_t *num, const char *str) {
+	double d = 0.0f;
+	sscanf (str, "%lf", &d);
```

Comment:
prefix sscanf with (void) to avoid warnings telling us that the return value is ignored